src/karma-webpack.js

```diff
@@ -198,7 +198,7 @@ Plugin.prototype.readFile = function(file, callback) {
   var doRead = function() {
     if (optionsCount > 1) {
       async.times(optionsCount, function(idx, callback) {
-        middleware.fileSystem.readFile(path.join(os.tmpdir(), '_karma_webpack_', idx, file.replace(/\\/g, '/')), callback)
+        middleware.fileSystem.readFile(path.join(os.tmpdir(), '_karma_webpack_', idx.toString(), file.replace(/\\/g, '/')), callback)
```

Comment:
`String(idx)` or coalesce it by changing it back to how it was handled [before](https://github.com/webpack-contrib/karma-webpack/commit/0616dda489d07f6a9bc3def1778dcef601411821#diff-1bc56a1b4b855af7b16012fcf5210059L200) https://github.com/webpack-contrib/karma-webpack/commit/0616dda489d07f6a9bc3def1778dcef601411821 (concat) please :)

Reply:
You got it